.github/workflows/rust.yml

```diff
@@ -1,30 +1,65 @@
 name: Rust
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
-  build:
+
+  codestyle:
     runs-on: ubuntu-latest
+    steps:
+      - name: Set up Rust
+        uses: hecrj/setup-rust-action@v1
+        with:
+          components: rustfmt
+          # Note that `nightly` is required for `license_template_path`, as
+          # it's an unstable feature.
+          rust-version: nightly
+      - uses: actions/checkout@v2
+      - run: cargo fmt -- --check --config-path <(echo 'license_template_path = "HEADER"')
```

Comment:
I just noticed this does not use `cargo +nightly fmt`, making it print a `Warning: can't set `license_template_path = "HEADER"`, unstable features are only available in nightly channel.` and presumably not run that check.

Reply:
Oh no, definitely an oversight on my part. Should be a trivial fix.